src/20_Components/amp-geo.html

```diff
@@ -0,0 +1,93 @@
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>amp-geo example</title>
+  <link rel="canonical" href="<%host%>/components/amp-geo/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <style amp-custom>
+    .flag { 
+      background: url("/img/american-flag.png") no-repeat; 
+      min-height: 16px;
+    }
+    .amp-iso-country-ca .flag { 
+      background: url("/img/canadian-flag.png") no-repeat;
+    }
+    .football:after { 
+      content: 'football';
+    }
+    .amp-geo-group-soccer .football:after { 
+      content: 'soccer' 
+    }
+  </style>
+  <!-- ## Setup -->
+  <!-- Import the `amp-geo` component -->
+  <script async custom-element="amp-geo" src="https://cdn.ampproject.org/v0/amp-geo-0.1.js"></script>
+  <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  
+</head>
+<body>
+  <!--
+     ## Basic usage
+  -->
+  <!--
+    `amp-geo` components is configured via json and every page can only contain one `amp-geo` tag. We are showing
+    different samples of `amp-geo` on this page and we will explain the json configuration for each sample.
+  -->
+  <amp-geo layout="nodisplay">
+      <script type="application/json">
+        {
+          "AMPBind": true,,
+          "ISOCountryGroups": {
+            "soccer": [ "au", "ca", "ie", "nz", "us", "za" ],
+            "football": [ "unknown" ]
+          }
+        }
+      </script>
+    </amp-geo>
+    <!-- `amp-geo` makes available a class `amp-iso-country-XX` where XX is the name of the country. 
+    In the following example, we determine if the user is in Canada, and if so, we display a maple leaf flag, otherwise the American flag. 
+    If you want to force the country to be Canada, you could append `#amp-geo=ca` to the URL like [https://ampbyexample.com/components/amp-geo#amp-geo=ca](https://ampbyexample.com/components/amp-geo#amp-geo=ca) and enable the development dev channel. 
+    Read more about debugging `amp-geo` on the official [doc](https://www.ampproject.org/docs/reference/components/amp-geo)-->
```

Comment:
Request change to:

<!-- `amp-geo` makes available a class `amp-iso-country-XX` where XX is an ISO Country Code. 
In the following example, the component determines the ISO Country Code corresponding to the user's approximate location. If this is "ca" (Canada), the page displays a maple leaf flag, otherwise the American flag. 
If you want to force the country to be Canada, you could append `#amp-geo=ca` to the URL like [https://ampbyexample.com/components/amp-geo#amp-geo=ca](https://ampbyexample.com/components/amp-geo#amp-geo=ca) and enable the development dev channel. 
Read more about debugging `amp-geo` on the official [doc](https://www.ampproject.org/docs/reference/components/amp-geo)-->